core/src/test/java/com/google/cloud/sql/core/CloudSqlInstanceTest.java

```diff
@@ -249,6 +294,77 @@ public void testCloudSqlRefreshesExpiredData() throws Exception {
         () -> instance.getSslData(TEST_TIMEOUT_MS) == data.getSslData(), 1000L);
   }
 
+  @Test
+  public void testThatForceRefreshBalksWhenAScheduledRefreshIsInProgress() throws Exception {
+    // Set expiration 1 minute in the future, so that it will trigger a scheduled refresh
+    // and won't expire during this testcase.
+    InstanceData aboutToExpireData =
+        new InstanceData(
+            null, new SslData(null, null, null), Instant.now().plus(1, ChronoUnit.MINUTES));
```

Comment:
What if we pulled this into a helper method that took the arguments of `plus`? From there we can make it more extensible for other tests.

Reply:
Done.